implement/io/input.go

```diff
@@ -1,28 +1,44 @@
 package iio
 
 import (
-	"errors"
 	"flag"
+	"fmt"
+	"os"
 	"strings"
 	"unicode/utf8"
 
 	sio "aman/struct/io"
+
 	"github.com/mattn/go-runewidth"
 	"github.com/nsf/termbox-go"
 )
 
 type InputStruct sio.InputStruct
 
+var (
+	versionFlag = flag.Bool("v", false, "show version")
+	helpFlag    = flag.Bool("h", false, "show help")
+)
+
+const (
+	optionText = "options arguments\n"
+	usageText  = "usage:\n" +
+		"  - Input `aman` and `YOUR_COMMAND` like `aman git`, `aman ls`, or `aman git status`\n" +
+		"  - Display options and you can select some one\n" +
+		"  - Output the integrated commands to your console\n"
```

Comment:
`$ go --help`とか`$ git --help`を参考に、実用例メインで記載する下記形式はどうでしょう？

```suggestion
	usageText  = "usage: aman <command> [arguments]\n" 
		"  <command>: command which has man result (ls, git,...)\n" +
		" [arguments]: arguments which command have ("status" for "git status")
```